apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformer.java

```diff
@@ -575,15 +592,15 @@ private PreAdjFissClaim transformMessageImpl(
         from::hasNpiNumber,
         from::getNpiNumber,
         to::setNpiNumber);
-    transformer.copyOptionalString(
-        namePrefix + PreAdjFissClaim.Fields.mbi, 1, 13, from::hasMbi, from::getMbi, to::setMbi);
-    transformer.copyOptionalString(
-        namePrefix + PreAdjFissClaim.Fields.mbiHash,
-        1,
-        64,
-        from::hasMbi,
-        () -> idHasher.computeIdentifierHash(from.getMbi()),
-        to::setMbiHash);
+    if (from.hasMbi()) {
+      final var mbi = from.getMbi();
+      if (transformer.validateString(namePrefix + PreAdjFissClaim.Fields.mbi, false, 1, 13, mbi)) {
```

Comment:
MBIs are 11 chars.  I know we've reserved 13 in the DB, but shouldn't validation still happen with 11?